samples/CognitiveSearch/CognitiveSearch/Generated/Models/FacetResult.cs

```diff
@@ -44,7 +44,7 @@ internal FacetResult(long? count, IReadOnlyDictionary<string, object> additional
         public IEnumerable<string> Keys => AdditionalProperties.Keys;
         /// <inheritdoc />
         public IEnumerable<object> Values => AdditionalProperties.Values;
-        /// <inheritdoc />
+        /// <inheritdoc cref="ICollection{T}.Count"/>
```

Comment:
looks like this needs to be distinguished between readonly and not. Will add the change.